File: P31_0709/P31_0709.c
#include<stdio.h>

int main()
{
    float l,b,h;

    //Taking innput from user
    printf("Enter the values of length, breadth and height of fridge:");
    scanf("%f%f%f",&l,&b,&h);

    //Checking the height of fridge first
    if(h>10)
    {
        printf("Fridge cannot be passed through the door");
    }

    else
    {
        if(l<=5 || b<=5)
        {
            printf("The volume of fridge is: %.2f",l*b*h);
        }

        else
        {
            printf("Fridge cannot be passed through the door");
        }
    }

    return 0;
}

File: P31_0709/P31_0709.py
l = float(input())
b = float(input())
h = float(input())

if(h > 10):
    print("Fridge cannot be passed through the door")
else:
    if (l <= 5 or b <= 5):
        print("The volume of the fridge is:", l*b*h)
    else:
        print("Fridge cannot be passed through door")


